completions/bash/skopeo

```diff
@@ -150,6 +154,7 @@ _skopeo_list_repository_tags() {
      --authfile
```

Comment:
Shouldn’t `skopeo layers’ have the option added as well?

Reply:
Yes, I think I missed that one because I made a search of the "--authfile" all over the files, and it is not in the `skopeo layers` completion either. So I am adding both the missing --authfile, and also I think the --no-creds option makes sense in here too.

Reply:
Thanks for the other fixes!